src/switch_rtp.c

```diff
@@ -2043,6 +2044,81 @@ static int using_ice(switch_rtp_t *rtp_session)
 	return 0;
 }
 
+static void sendRTCPEvent(switch_rtp_t *rtp_session ,struct switch_rtcp_sender_report *sr,struct switch_rtcp_report_block *rtcp_report_block) {
```

Comment:
bad naming. see the other function names in switch_rtp.c

Reply:
@dragos-oancea please review our update.